changelog.d/OPW-86.fix.md

```diff
@@ -0,0 +1,2 @@
+Fix an issue where the `datadog_logs` sink could produce a request larger than the allowed API
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** 
‏
```suggestion
Fixed an issue where the `datadog_logs` sink could produce a request larger than the allowed API
```